test/l4lb/test.sh

```diff
@@ -4,7 +4,7 @@ set -eux
 
 IMG_OWNER=${1:-cilium}
 IMG_TAG=${2:-latest}
-PULL_REQUEST_CHECKOUT=${3:-../..}
+HELM_CHART_DIR=${3:-/vagrant/pull-request/install/kubernetes/cilium}
```

Comment:
For backward compatibility: 
```suggestion
HELM_CHART_DIR=${3:-/vagrant/install/kubernetes/cilium}
```